docs/references/nextjs/read-sessions-data.mdx

```diff
@@ -0,0 +1,240 @@
+---
+title: Read session and user data with Clerk and Next.js
+description: Learn how to use Clerk to quickly and easily add secure authentication and user management to your Next.js application. Clerk works seamlessly on both client side and server side components.
+---
+
+# Read session and user data with Clerk and Next.js
```

Comment:
consistent with the other guides.
note: if we update this h1, we need to update the title in the frontmatter to match.

```suggestion
# Read session and user data in your Next.js App with Clerk
```